src/main/java/net/dv8tion/jda/api/interactions/Interaction.java

```diff
@@ -42,6 +42,8 @@
  *     <br>Which supports direct message edits and deferred message edits (or no-operation) via {@link IMessageEditCallback#editMessage(String)} and {@link IMessageEditCallback#deferEdit()}</li>
  *     <li>{@link IAutoCompleteCallback}
  *     <br>Which supports choice suggestions for auto-complete interactions via {@link IAutoCompleteCallback#replyChoices(Command.Choice...)}</li>
+ *     <li>{@link net.dv8tion.jda.api.interactions.callbacks.IModalCallback}
+ *     <br>Which supports replying using a {@link net.dv8tion.jda.api.interactions.components.text.Modal Modal} via {@link net.dv8tion.jda.api.interactions.callbacks.IModalCallback#replyModal(net.dv8tion.jda.api.interactions.components.text.Modal)}</li>
```

Comment:
You can use imports here instead of qualified names to make it more readable, similar to the others in the list.